circle.yml

```diff
@@ -6,7 +6,9 @@ machine:
 
 dependencies:
   override:
+    - brew install swiftlint
     - bundle install
+    - rake danger
```

Comment:
I've roiled back to before I started messing about with env variables, this branch could be good to test https://github.com/danger/danger/pull/292 when ready.